src/audio/heap.h

```diff
@@ -64,9 +70,14 @@ struct SoundMultiPool {
     /*     */ u32 pad2[4];
 }; // size = 0x1D0
 
+#ifdef VERSION_SH
```

Comment:
why is this ifdefed to shindou?

Reply:
It's unused in the other versions, and primarily it just means darkening their displays in VSCode

Reply:
fair, looks good then